packages/synapse-react-client/src/lib/containers/FileUpload.tsx

```diff
@@ -1,19 +1,27 @@
 import * as React from 'react'
-import { Button } from 'react-bootstrap'
+import { Button, ButtonProps } from '@mui/material'
 import { uploadFile } from '../utils/SynapseClient'
 import { useSynapseContext } from '../utils/SynapseContext'
 import { FileUploadComplete, UploadCallbackResp } from '../utils/synapseTypes'
 
 export type FileUploadProps = {
   id?: string
   label?: string
-  variant?: string // allow you to change the appearance of the button to link (see react bootstrap doc)
+  variant?: ButtonProps['variant']
```

Comment:
You could drop this prop and just include `variant` in the `buttonProps` object. You might need to add a default `buttonProps` value where `variant` is defined (I think it's a required prop). Up to you.

Reply:
I originally included `variant` in `buttonProps`, but ended up moving it into `FileUploadProps` so it would be easier to switch between / see the possible variants in Storybook. I defer to your experience though -- if you think it fits better to group with the other `buttonProps`, then I'm happy to move it into `buttonProps`!